src/FilterButtonList.js

```diff
@@ -0,0 +1,139 @@
+import F from 'futil-js'
+import _ from 'lodash/fp'
+import React from 'react'
+import { observer } from 'mobx-react'
+import {
+  Dynamic,
+  Flex,
+  CheckButton as DefaultCheckButton,
+  Modal as DefaultModal,
+  Popover as DefaultPopover,
+} from './layout'
+import DefaultIcon from './DefaultIcon'
+import InjectTreeNode from './utils/injectTreeNode'
+import styles from './styles'
+
+export let DefaultMissingTypeComponent = InjectTreeNode(({ node = {} }) => (
+  <div>
+    Type <b>{node.type}</b> is not supported (for key <i>{node.key}</i>)
+  </div>
+))
+
+let FieldDescription = observer(({ node }) => (
+  <div className="filter-description">
+    {node.fieldDescription} {node.typeDescription}
```

Comment:
We should avoid putting these on `node` because they're display only and don't need to be sent to the server or persisted in the DB. you can either drive these as props to FilterButtonItem (which can be set via something like `mapNodeToProps`) or look up description off of the `field` from the schema an override `description` by simply changing the type components that are passed in at the end user level

Reply:
These were supposed to be placeholders for the functionality you're describing - it completely slipped my mind that I hadn't implemented it yet 🤦‍♀

For the wizard stuff, I think we'll need all of the above for maximum granularity. So `label`/`description` props from `mapNodeToProps` take precedence (for specific overrides), then the value of `mapNodeToLabel`/`mapNodeToDescription`, then various fallbacks like `node.field` (for the label). 

This is particularly relevant for the description stuff; we'll need to be able to knit looked-up field and type descriptions together in a friendly-enough way, and `mapNodeToDescription` allows us to do that while still having the option to set descriptions on a per-node basis with `mapNodeToProps`.